spec/migrations/20190726204302_remove_local_default_embedded_ansible_repos_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_migration
+
+describe RemoveLocalDefaultEmbeddedAnsibleRepos do
+  let(:configuration_script_source_stub) { migration_stub(:ConfigurationScriptSource) }
+  let(:git_repository_stub)              { migration_stub(:GitRepository) }
+  let(:embedded_ansible_source_type)     { "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource" }
+  let(:ansible_tower_source_type)        { "ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScriptSource" }
+
+  migration_context :up do
```

Comment:
Would you mind adding a test for a ConfigurationScriptSource that doesn't match the `scm_url`s?

Reply:
I have one here https://github.com/ManageIQ/manageiq-schema/pull/400/files#diff-4b8a158a4b41502e5e406302cb471f9cR46-R56

Reply:
Oh nevermind that's type
I'll add one